templates/resources/vrack_dedicated_cloud_datacenter.md.tmpl

```diff
@@ -0,0 +1,34 @@
+---
+subcategory : "vRack"
+---
+
+{{/* This template serves as a starting point for documentation generation, and can be customized with hardcoded values and/or doc gen templates.
+
+For example, the {{ .SchemaMarkdown }} template can be used to replace manual schema documentation if descriptions of schema attributes are added in the provider source code. */ -}}
+
+# ovh_vrack_dedicated_cloud_datacenter
+
+Move a Dedicated Cloud Datacenter to a vrack.
+
+## Example Usage
+
```

Comment:
you should mention before this example that the resource cannot be created and needs to be imported first. I know the information is already in the `## Import` section, but a paragraph starting with `~> ` would be more visible.